go/install/update.go

```diff
@@ -234,12 +248,17 @@ func (u *Updater) checkUpdate(sourcePath string, destinationPath string) error {
 
 func (u *Updater) apply(src string, dest string) error {
 	if _, err := os.Stat(dest); err == nil {
-		tmpDirName, err := libkb.TempFile(fmt.Sprintf("%s.", path.Base(dest)))
+		tmpFileName, err := libkb.TempFileName(fmt.Sprintf("%s.", path.Base(dest)))
 		if err != nil {
 			return err
 		}
-		u.G().Log.Info("Moving (existing) %s to %s", dest, tmpDirName)
-		err = os.Rename(dest, tmpDirName)
+		tmpPath := path.Join(os.TempDir(), "KeybaseBackup", tmpFileName)
+		err = libkb.MakeParentDirs(tmpPath)
+		if err != nil {
+			return err
+		}
+		u.G().Log.Info("Moving (existing) %s to %s", dest, tmpPath)
+		err = os.Rename(dest, tmpPath)
```

Comment:
Have an open ticket for this here:
https://keybase.atlassian.net/browse/DESKTOP-165

Putting it in a subdir to make it easier to clear